packages/backend/package.json

```diff
@@ -35,6 +35,7 @@
     "@backstage/plugin-adr-backend": "workspace:^",
     "@backstage/plugin-app-backend": "workspace:^",
     "@backstage/plugin-auth-backend": "workspace:^",
+    "@backstage/plugin-auth-backend-module-oauth2-proxy-provider": "workspace:^",
```

Comment:
this shouldn't be needed, right? 🤔 

```suggestion
```

Reply:
Fixed in https://github.com/backstage/backstage/pull/20685/commits/f88b2da6c9741d35234d31cdfef9e305a4f0b624